opencga-app/app/scripts/docker/iva/Dockerfile

```diff
@@ -1,8 +1,5 @@
 FROM httpd
 
-# Minimum compatible git commit, please only use later commits.
-ARG IVA_COMMIT_SHA="dce421b6933d540d9c511c72e63c0e85dff00667"

```

Comment:
why remove and not just change the git SHA?

Reply:
I guess that would work too, but as a new IVA release is made, want the image to contain the laltest changes? Otherwise going to have to keep updating the git SHA. Should be built from CI on the IVA repo...

Reply:
ideally this Dockerfile would live inside the IVA repo - that way you could build a docker image for any version by just changing around in git before building... However, as this doesn't it means you have no control of which revision of IVA to build from the remote endpoint you'll always just get whatever latest is on IVA master, if it's broken or not.

Reply:
probably a case of YAGNI on my behalf just prefer flexibility rather than rigidness

Reply:
But I think development is done on develop, and master is "stable"?